### PR TITLE
Create replica set form

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- Create replica set form

**Related issue(s)**

### CODE REVIEW

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -1110,13 +1110,24 @@ preferences:
...
   cpu: CPU
+  create-modal:
+    labels:
+      min-ready-seconds: Minimum ready seconds
+      replicas: Replicas
+    placeholders:
+      min-ready-seconds: Minimum number of seconds
+      replicas: Replicas
+    tooltips:
+      min-ready-seconds: Minimum number of seconds for which a newly created pod should be ready without any of its container crashing, for it to be considered available.
```
Comment (grego952) [inline core/src/i18n/en.yaml:1121]:
```suggestion
      min-ready-seconds: Minimum number of seconds for which a newly created Pod should be ready without any of its container crashing, to be considered available.
```

Comment (alexandra-simeonova) [inline core/src/i18n/en.yaml:1121]:
```suggestion
      min-ready-seconds: Minimum number of seconds for which a newly created Pod should be ready without any of its containers crashing in order to be considered available.
```

**Review by alexandra-simeonova — APPROVED** [review]

File: core-ui/src/shared/components/K8sResourceSelect.js [hunk 2]
```diff
@@ -98,6 +98,7 @@ export function K8sResourceSelect({
         })}
```
Comment (dariadomagala) [inline core-ui/src/shared/components/K8sResourceSelect.js:98]: I cannot add a comment there but shouldn't we switch the selectionType to 'manual' (line 93)? this one doesn't work good

**Review by dariadomagala — APPROVED** [review]

**Review by alexandra-simeonova — APPROVED** [review]

**Pull request merged**